### ISSUE TITLE
Rename `initIndex` into `initialIndex` in useGalleryPreview()

### ISSUE THREAD

**Issue by Dominic-Preap** [chrizuuu/react-native-gallery-preview#16]
It would be nice to have `galleryPreviewState` from `useGalleryPreview` has the same props name as in `ImageGallery`  so we can spread props inside. I stuck half day not seeing the diff index name.

`isVisible` is already the same, why not `initialIndex` & `initIndex` cannot be the same.

```ts
export default function Example() {
  const { galleryPreviewState, closeGalleryPreview, openGalleryPreview } = useGalleryPreview();
  return (
    <ImageGallery
      images={[]}
      onRequestClose={closeGalleryPreview}
      isVisible={galleryPreviewState.isVisible}
      initialIndex={galleryPreviewState.initIndex} // ! typo props here
    />
  );
}

```

**Comment by chrizuuu** [comment]
hey, good point, I must have missed that. I'll put a patch on it over the weekend.

**Comment by chrizuuu** [comment]
fixed [1.3.2](https://github.com/chrizuuu/react-native-gallery-preview/releases/tag/v1.3.2)
